dialect/window.py

```diff
@@ -767,6 +789,22 @@ def ui_clear(self, _action, _param):
         self.src_buffer.props.text = ''
         self.src_buffer.emit('end-user-action')
 
+    def set_font_size(self, size):
+        self.font_size = size
+        self.font_css_provider.load_from_data(
+            "textview { font-size: "
+            + str(size)
+            + "pt; }"
+        )
```

Comment:
Let's make it a bit more compact

```suggestion
        self.font_css_provider.load_from_data(f'textview {{ font-size: { str(size) }pt; }}' )
```